src/blockedtuple.jl

```diff
@@ -23,6 +23,7 @@ end
 
 # Base interface
 Base.axes(bt::AbstractBlockTuple) = (blockedrange([blocklengths(bt)...]),)
+Base.axes(::AbstractBlockTuple{0}) = (blockedrange(zeros(Int, 0)),)
```

Comment:
```suggestion
Base.axes(::AbstractBlockTuple{0}) = (blockedrange(Int[]),)
```
Just a bit easier to understand what the intent is.